packages/theme-ui/README.md

```diff
@@ -116,7 +116,7 @@ scales help you build UI rooted in constraint-based design principles.
 npm install theme-ui @emotion/react
 ```
 
-_If you don't need color modes, components and MDX support, you can install
+_If you don't need color modes, components, or MDX support, you can install
```

Comment:
```suggestion
_If you don't need color modes or components you can install
```

I missed this when I removed MDX from `theme-ui`. Everyone using it will now need to directly import `@theme-ui/mdx`.